pootle/apps/accounts/management/commands/__init__.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth import get_user_model
+
+
+class UserCommand(BaseCommand):
+    """Base class for handling user commands."""
+
+    args = "user"
+
+    def create_parser(self, prog_name, subcommand):
+        self.prog_name = prog_name
+        self.subcommand = subcommand
+        return super(UserCommand, self).create_parser(prog_name, subcommand)
+
+    def get_user(self, username):
+        User = get_user_model()
```

Comment:
I suggest putting this line in the module (outside of `UserCommand` class), so it is assigned just once.